Client/Application/WebServer.swift

```diff
@@ -18,8 +18,10 @@ class WebServer {
     }
 
     func start() -> Bool {
-        if !server.running {
-            try! server.startWithOptions([GCDWebServerOption_Port: 6571, GCDWebServerOption_BindToLocalhost: true, GCDWebServerOption_AutomaticallySuspendInBackground: true])
+        withExtendedLifetime(server) {
```

Comment:
Agree with @sleroux  .. I wrote that code in the Swift 1.0 era and I don't know if the way we do the shared instance of `WebServer` is correct now actually. Can we try this first:

```
class WebServer {
    static let sharedInstance = WebServer()
}
```
That is the documented way to do singleton instances. I don't think there is a reason that instance can ever be deallocated.